docs/samples/python/customizing_presidio_analyzer.ipynb

```diff
@@ -0,0 +1,677 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "jupyter:\n",
+    "  jupytext:\n",
+    "    text_representation:\n",
+    "      extension: .md\n",
+    "      format_name: markdown\n",
+    "      format_version: '1.1'\n",
+    "      jupytext_version: 1.1.0\n",
+    "  kernelspec:\n",
+    "    display_name: Python 3\n",
+    "    language: python\n",
+    "    name: python3\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customizing the PII analysis process in Microsoft Presidio\n",
+    "\n",
+    "This notebooks covers different customization use cases to:\n",
+    "1. Adapt Presidio to detect new types of PII entities\n",
+    "2. Adapt Presidio to detect PII entities in a new language\n",
+    "3. Embed new types of detection modules into Presidio, to improve the coverage of the service."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation\n",
+    "First, let's install presidio using `pip`. For detailed documentation, see the [installation docs](https://microsoft.github.io/presidio/installation).\n",
+    "\n",
+    "Install from PyPI:\n",
+    "```sh\n",
+    "pip install presidio_analyzer\n",
+    "pip install presidio_anonymizer\n",
+    "python -m spacy download en_core_web_lg\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting started\n",
+    "\n",
+    "The high level process in Presidio-Analyzer is the following:\n",
```

Comment:
![Screen Shot 2021-03-11 at 13 43 47](https://user-images.githubusercontent.com/1862328/110782490-ddda8b00-826f-11eb-8f99-0a9bd05a5b13.png)
Image is broken and should the numbers be there? e.g. In [23]: